src/main/scala/io/iohk/ethereum/ledger/Ledger.scala

```diff
@@ -2,13 +2,14 @@ package io.iohk.ethereum.ledger
 
 import io.iohk.ethereum.db.storage.NodeStorage
 import io.iohk.ethereum.domain._
+import io.iohk.ethereum.network.p2p.messages.PV63.Receipt
 import io.iohk.ethereum.utils.{Config, Logger}
 import io.iohk.ethereum.vm.{GasFee, _}
 
 object Ledger extends Logger {
 
   type PR = ProgramResult[InMemoryWorldStateProxy, InMemoryWorldStateProxyStorage]
-  type ExecResult = (InMemoryWorldStateProxy, BigInt)
+  type ExecResult = (InMemoryWorldStateProxy, BigInt, Seq[Receipt])
```

Comment:
do we expect any more values here in the future? If so maybe it's good time to make it a `case class`